.all-contributorsrc

```diff
@@ -1819,6 +1832,11 @@
       "contributions": [
         "doc"
       ]
-    }
-  ]
+    },
```

Comment:
where did this block come from? does not seem related to the PR

Reply:
seems to be a merge accident, the same is also at the top.

Please make sure your fork is always up-to-date.